src/languages/es-ES.ts

```diff
@@ -2260,6 +2266,37 @@ export default class extends Language {
 			extendedHelp: `This command shows a random starred message or the starboard usage and leaderboard for this server.`
 		}),
 
+		/**
+		 * ##################
+		 * SUGGESTIONS COMMANDS
+		 */
+
+		COMMAND_SUGGEST_DESCRIPTION: 'Manda una sugerencia para el servidor.',
+		COMMAND_SUGGEST_EXTENDED: builder.display('suggest', {
+			extendedHelp: 'Publica una recomendación en el canal de recomendaciones del servidor',
+			explainedUsage: [
+				['suggestion', 'Su recomendación']
+			],
+			examples: [
+				'Crear un canal de música.'
+			],
+			reminder: 'Necesita tener configurado un canal de sugerencias para que este comando funcione. Si eres un administrador, te preguntaré si quieres configurarlo al momento de su uso.'
+		}),
+		COMMAND_RESOLVESUGGESTION_DESCRIPTION: 'Modifica el estado de la sugerencia.',
+		COMMAND_RESOLVESUGGESTION_EXTENDED: builder.display('resolvesuggestion', {
+			extendedHelp: `Este comando marca una recomendación como aceptada, considerada o denegada.
+			También se puede configurar para enviar un mensaje directo al autor con respecto al estado de su recomendación, con la configuración \`suggestions.on-action.dm\`.
+			Además, en caso de que desee preservar el anonimato, puede ocultar su nombre utilizando la configuración \`suggestions.on-action\`, que puede anularse con las \`--hide-author\` y \`--show-author\` banderas`,
```

Comment:
We use "señal" (signal) as a translation for "flag" in this specific context. Maybe we should adopt it everywhere else?
```suggestion
			Además, en caso de que desee preservar el anonimato, puede ocultar su nombre utilizando la configuración \`suggestions.on-action\`, que puede anteponerse con las señales \`--hide-author\` y \`--show-author\`.`,
```